src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -246,7 +246,7 @@ private static Version GetICUVersion()
             int version = 0;
             try
             {
-                Type interopGlobalization = Type.GetType("Interop+Globalization");
+                Type interopGlobalization = Type.GetType("Interop+Globalization, System.Private.CoreLib");
```

Comment:
The Interop type exist both in the current assembly and in CoreLib. Apparently we're supposed to try in both. The reflection stack is not structured to fix this in straighforward way. I just really don't care.